frontend/components/notifications/management/NotificationsManagementDashboards.vue

```diff
@@ -0,0 +1,330 @@
+
+const { groups } = useValidatorDashboardGroups()
+
+const { width } = useWindowSize()
+const colsVisible = computed(() => {
+  return {
+    networks: width.value > 1101,
+    webhook: width.value >= 945,
+    subscriptions: width.value >= 725
+  }
+})
+
+const loadData = (query?: TableQueryParams) => {
+  if (!query) {
+    query = { limit: pageSize.value, sort: 'dashboard_id:desc' }
+  }
+  setQuery(query, true, true)
+}
+
+watch(query, (q) => {
+  getDashboards(q)
+}, { immediate: true })
+
+const groupNameLabel = (groupId?: number) => {
+  return getGroupLabel($t, groupId, groups.value, 'Σ')
+}
+
+const onSort = (sort: DataTableSortEvent) => {
+  loadData(setQuerySort(sort, lastQuery.value))
+}
+
+const setCursor = (value: Cursor) => {
+  cursor.value = value
+  loadData(setQueryCursor(value, lastQuery.value))
+}
+
+const setPageSize = (value: number) => {
+  pageSize.value = value
+  loadData(setQueryPageSize(value, lastQuery.value))
+}
+
+const setSearch = (value?: string) => {
+  loadData(setQuerySearch(value, lastQuery.value))
+}
+
+const wrappedDashboards = computed(() => {
+  if (!dashboards.value) {
+    return
+  }
+  return {
+    paging: dashboards.value.paging,
+    data: dashboards.value.data.map(d => ({ ...d, identifier: `${d.dashboard_type}-${d.dashboard_id}-${d.group_id}` }))
+  }
+})
+
+const onEdit = (col: 'delete' | 'subscriptions' | 'webhook' | 'networks', row: NotificationsManagementDashboardRow) => {
+  switch (col) {
+    case 'subscriptions':
+      alert('TODO: edit subscriptions' + row.group_id)
+      break
+    case 'webhook':
+      alert('TODO: edit webhook' + row.group_id)
+      break
+    case 'networks':
+      alert('TODO: edit networks' + row.group_id)
+      break
+    case 'delete':
+      alert('TODO: delete' + row.group_id)
+      break
+  }
+}
+
+</script>
+
+<template>
+  <div>
+    <Teleport to="#notifications-management-search-placholder">
+      <BcContentFilter :search-placeholder="$t('placeholder')" class="search" @filter-changed="setSearch" />
+    </Teleport>
+
+    <ClientOnly fallback-tag="span">
+      <BcTable
+        :data="wrappedDashboards"
+        data-key="identifier"
+        :expandable="!colsVisible.networks"
+        class="notifications-management-dashboard-table"
+        :cursor="cursor"
+        :page-size="pageSize"
+        :selected-sort="tempQuery?.sort"
+        :loading="isLoading"
+        @set-cursor="setCursor"
+        @sort="onSort"
+        @set-page-size="setPageSize"
+      >
+        <Column
+          field="dashboard_id"
+          body-class="dashboard-col"
+          header-class="dashboard-col"
+          :sortable="true"
+          :header="$t('notifications.col.dashboard')"
+        >
+          <template #body="slotProps">
+            <span>
+              {{ slotProps.data.dashboard_name }}
+            </span>
+          </template>
+        </Column>
```

Comment:
**Bug:** Compared to the **Design**,
![image](https://github.com/gobitfly/beaconchain/assets/114066135/aa6411c1-4831-46dd-93fc-00a453241738)
the **Implementation** is missing the dashboard type icon
![image](https://github.com/gobitfly/beaconchain/assets/114066135/51993d77-8aa1-4263-a776-71913423db8f)